### PR TITLE
feat(artisthub): wip merge sync #2

### PR DESCRIPTION
**Pull request by So1ene**
before: https://main--stock--adobecom.hlx.page/pages/artisthub/
after: https://artisthub-3-1--stock--webistry-development.hlx.page/pages/artisthub/

### CODE REVIEW

File: pages/scripts/scripts.js [hunk 1]
```diff
@@ -53,9 +59,13 @@ const { loadArea,  loadDelayed,  setConfig } = await import(`${miloLibs}/utils/u
 
 (async function loadPage() {
   setConfig({ ...CONFIG, miloLibs });
+  document.body.style.visibility = 'hidden';
```
Comment (auniverseaway) [inline pages/scripts/scripts.js:62]: You shouldn't need this. The body is already being hidden in head.html.

Comment (So1ene) [reply pages/scripts/scripts.js:62]: It seems to appear before the Template scripts/styles if I don't have this

File: pages/scripts/utils.js [hunk 2]
```diff
@@ -145,6 +128,53 @@ export function decorateButtons() {
...
+  if (url.host === 'youtu.be') vid = url.pathname.substr(1);
+  $video.innerHTML = /* html */`
+  <div class="youtube-container">
+    <iframe src="https://www.youtube.com/embed/${vid}?rel=0&amp;modestbranding=1&amp;playsinline=1&amp;autohide=1&amp;showinfo=0&amp;rel=0&amp;controls=1&amp;autoplay=1&amp;mute=1&amp;loop=1&amp;playlist=${vid}" frameBorder="0" allowfullscreen="" scrolling="no" allow="encrypted-media; accelerometer; gyroscope; picture-in-picture; autoplay" title="content from youtube" loading="lazy"></iframe>
+  </div>
+  `;
+  return $video;
+}
+
+export function unwrapFragments() {
```
Comment (auniverseaway) [inline pages/scripts/utils.js:148]: Are you finding that you want this for all fragments? I see many scenarios where you want a fragment inside a block (like in columns) and unwrapping would not have the desired result.

Comment (So1ene) [reply pages/scripts/utils.js:148]: I think this would apply only to fragments that are on their own, not inside a block

**Pull request merged**